hoodie-client/src/main/java/com/uber/hoodie/config/HoodieCompactionConfig.java

```diff
@@ -42,7 +42,7 @@
 
   // Turn on inline compaction - after fw delta commits a inline compaction will be run
   public static final String INLINE_COMPACT_PROP = "hoodie.compact.inline";
-  private static final String DEFAULT_INLINE_COMPACT = "true";
+  private static final String DEFAULT_INLINE_COMPACT = "false";
```

Comment:
lets leave this as true? we can change it to false internally, true probably is less confusing for a new user anyway

Reply:
done

Reply:
@vinothchandar Setting this to false again since HoodieCopyOnWriteTable existing configuration in jobs will break since now I throw an exception in the table.compact() for COW, earlier we just silently let it pass.

Reply:
can we make this a No-op instead of throwing exception? You can look at it as - COW table is always fully compacted, so no work to do. 
It will be good to leave the default to true, and do above if thats the only issue here

Reply:
I see, you also have smething to return.. lets leave this as is then and revist.,